Config/ProjectApollo/RTCC/EMPs/EMP SL-55.txt

```diff
@@ -0,0 +1,14 @@
+IMU slip ring program

```

Comment:
Nitpick: most of the rest of the titles are Title Case, so for consistency this would be "IMU Slip Ring Program"